wiki/configuration.md

```diff
@@ -296,3 +296,13 @@ Eiffel-Intelligence will decrypt password properties with help of the provided s
 Execute Eiffel-Intelligence with "jasypt.encryptor.password=mySecretEncryptorPassword" flag, example:
 
         java -jar eiffel-intelligence-<version>.war --jasypt.encryptor.password=mySecretEncryptorPassword --spring.config.location=/path/to/application.properties
+
+## Removal of unused bindings from RabbitMQ using mongoDB
```

Comment:
I want this section of the documentation to be placed together with the other RabbitMQ configuration documentation.

Can the header be something like: "**Storage of rabbit MQ binding keys in mongo database**"? Removal could imply that a user is responsible for this, which is confusing.